frontend/src/assets/stylesheets/components/post.sass

```diff
@@ -1,3 +1,8 @@
+$animate: all 0.2s .1s ease-in-out;
```

Comment:
Specify what will be animate, avoid animating all elements it might occur unexpected behaviour